shapes/neurosciencegraph/datashapes/atlas/imagestack/schema.json

```diff
@@ -1,138 +1,143 @@
-      ]
-    }
-  ]
-}
\ No newline at end of file
+  {
+    "@context": [
+      "https://incf.github.io/neuroshapes/contexts/schema.json",
+      {
+        "this": "https://neuroshapes.org/dash/imagestack/shapes/"
+      }
+    ],
+    "@id": "https://neuroshapes.org/dash/imagestack",
+    "@type": "nxv:Schema",
+    "import": [
+      "https://neuroshapes.org/commons/transformableobject",
+      "https://neuroshapes.org/commons/quantitativevalue"
+    ],
+    "prov:wasDerivedFrom": "https://github.com/INCF/neuroshapes/blob/v0.3.15/modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/imagestack/v1.0.0.json",
+    "shapes": [
+      {
+        "@id": "this:ImageStackShape",
+        "@type": "sh:NodeShape",
+        "label": "Image stack",
+        "nodeKind": "sh:BlankNodeOrIRI",
+        "targetClass": "nsg:ImageStack",
+        "and": [
+          {
+            "node": "https://neuroshapes.org/commons/transformableobject/shapes/TransformableObjectShape"
+          },
+          {
+            "property": [
+              {
+                "path": "nsg:imageModality",
+                "name": "Image modality",
+                "description": "Modality of the image stack",
+                "datatype": "xsd:string",
+                "maxCount": 1
+              },
+              {
+                "path": "nsg:slicingPlane",
+                "name": "Slicing plane",
+                "description": "Slicing plane of the brain",
+                "in": [
+                  "Sagittal",
+                  "Para-sagittal",
+                  "Coronal",
+                  "Horizontal"
+                ],
+                "maxCount": 1
+              },
+              {
+                "path": "nsg:sliceWidth",
+                "name": "Slice width",
+                "description": "Width of the 2D slice",
+                "datatype": "xsd:integer",
+                "maxCount": 1
+              },
+              {
+                "path": "nsg:sliceHeight",
+                "name": "Slice height",
+                "description": "Height of the 2D slice",
+                "datatype": "xsd:integer",
+                "maxCount": 1
+              },
+              {
+                "path": "nsg:sliceResolution",
+                "name": "Slice resolution",
+                "description": "Resolution of the 2D slice",
+                "node": "this:SliceResolutionShape",
+                "maxCount": 1
+              },
+              {
+                "path": "nsg:numberOfSlices",
+                "name": "Number of slices",
+                "description": "Number of slices of the image stack",
+                "datatype": "xsd:integer",
+                "maxCount": 1
+              },
+              {
+                "path": "nsg:sliceInterval",
+                "name": "Slice interval",
+                "description": "Slice interval",
+                "node": "this:SliceIntervalShape",
+                "maxCount": 1
+              }
```

Comment:
We might want to add a better description of this property. I assume it refers to the interval at which individual slices in a stack of slices were sampled by the imaging method.

Reply:
please do.